WORKSPACE

```diff
@@ -152,6 +152,17 @@ maybe(
     path = "third_party/tensorflow",
 )
 
+# TF depends on tf_toolchains.
+http_archive(
+    name = "tf_toolchains",
+    sha256 = "4c4231037088ad3b56520712a370db45860100304eea2c1a2dfc9944db10da4f",
+    strip_prefix = "toolchains-1.1.3",
+    urls = [
+        "http://mirror.tensorflow.org/github.com/tensorflow/toolchains/archive/v1.1.3.tar.gz",
+        "https://github.com/tensorflow/toolchains/archive/v1.1.3.tar.gz",
+    ],
+)
```

Comment:
Did you want to get the rollback of this code in before merging this to `main`?

Reply:
I'll repush it in. I wanted to check whether it was passing to this point first.

Reply:
Nope. Looks like that change causes some more issues. Submitted this first, then I can fix up the other issues.